pkg/admission/server_test.go

```diff
@@ -455,6 +455,144 @@ func TestServeHTTPSubmissions(t *testing.T) {
 				wantRespCode:       http.StatusInternalServerError,
 				wantFailureMessage: "unknown resource 'brokenroutes'\n",
 			},
+			{
+				name: "valid v1beta2 HTTPRoute resource with timeouts",
```

Comment:
Why are these timeout tests needed in server_test.go? Are they somehow needed to test that timeouts also work in v1alpha2 resources?

Anyway, if they are needed I think the name is wrong: v1beta2 -> v1alpha2?
```suggestion
				name: "valid v1alpha2 HTTPRoute resource with timeouts",
```

Reply:
this was wrongly added, thanks @frankbu - addressed with 482ce1559fb95af38b191cef5eb30f9bfd638e54